recipes/traja/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "traja" %}
+{% set version = "0.0.2a2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d13d54d66887b0c2de1db6d5161d72e96f8411364364e6b442e7e5f341f09a5f
+
+build:
+  number: 1
```

Comment:
Build numbers should start at 0 for new versions

Reply:
Done, thanks